src/main/java/edu/harvard/iq/dataverse/api/Datasets.java

```diff
@@ -3282,4 +3286,107 @@ public Response getCurationStates() throws WrappedResponse {
         csvSB.append("\n");
         return ok(csvSB.toString(), MediaType.valueOf(FileUtil.MIME_TYPE_CSV), "datasets.status.csv");
     }
+    
+    //APIs to manage archival status
+    
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    @Path("/submitDatasetVersionToArchive/{id}/{version}/status")
+    public Response getDatasetVersionToArchiveStatus(@PathParam("id") String dsid,
+            @PathParam("version") String versionNumber) {
+
+        try {
+            AuthenticatedUser au = findAuthenticatedUserOrDie();
+            if (!au.isSuperuser()) {
+                return error(Response.Status.FORBIDDEN, "Superusers only.");
+            }
+            Dataset ds = findDatasetOrDie(dsid);
+           
+            DatasetVersion dv = datasetversionService.findByFriendlyVersionNumber(ds.getId(), versionNumber);
```

Comment:
Not sure I saw it but looking now, getDatasetVersionOrDie doesn't support the friendlyVersionNumber syntax which is a ~requirement here (that's the convention used in the Bag naming and metadata that the archiver gets). I can go ahead and add parsing for that which would have the presumably useful side effect of letting other datasetversion api calls support the friendly version number as well.

Reply:
It should. I'm seeing `handleSpecific(long major, long minor)`. It's used by https://guides.dataverse.org/en/5.11/api/native-api.html#get-version-of-a-dataset which has a "friendly" example of "1.0".

Reply:
Hmm - calls to this are counted with MakeDataCounts. I guess since these are API calls they should count? (although they are clearly system-level interactions and not end-user interaction with the data). In any case, I went ahead for now.

Reply:
I dunno. I'd leave this out of Make Data Count. Like you said, these are systems setting and retrieving archival status messages. The spirit of Make Data Count is views/investigations and downloads/requests. People and machines looking at data.